src/main/scala/no/ndla/audioapi/service/search/AudioIndexService.scala

```diff
@@ -49,10 +53,12 @@ trait AudioIndexService {
           keywordField("license"),
           keywordField("defaultTitle"),
           textField("authors").fielddata(true),
-          keywordField("audioType")
+          keywordField("audioType"),
+          nestedField("series").fields(seriesIndexService.seriesIndexFields)
         ) ++
           generateLanguageSupportedFieldList("titles", keepRaw = true) ++
-          generateLanguageSupportedFieldList("tags")
+          generateLanguageSupportedFieldList("tags") ++
+          generateLanguageSupportedFieldList("manuscript")
```

Comment:
Burde forsåvidt også kunne søke i podcastMeta.introduction. Det kjem dei heilt klart til å etterlyse.

Reply:
Skal se hva jeg klarer å finne på :+1:

Reply:
Flyttet podcastMeta.introduction til et eget felt for å slippe å ha en "unødvendig" `nestedQuery` i alle søk med query.
Om koden blir for lite lesbar så går det an å gjøre [sånn her](https://github.com/NDLANO/audio-api/compare/searchable-manuscript...is-this-cooler) istedet, men jeg tror det er litt uheldig ytelsesmessig.